tst/EnergyPlus/unit/AirTerminalSingleDuct.unit.cc

```diff
@@ -682,4 +683,180 @@ TEST_F(EnergyPlusFixture, AirTerminalSingleDuctVAVReheat_NormalActionTest)
+    auto &thisAirTerminalOutlet = SingleDuct::SysOutlet(SysNum);
+
+    // check VAV reheat VS Fan air terminal inputs
+    EXPECT_EQ("AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan", thisAirTerminal.SysType);
+    EXPECT_EQ("VAV RHT VS FAN AIRTERM", thisAirTerminal.SysName); 
+    EXPECT_EQ("COIL:HEATING:ELECTRIC", thisAirTerminal.ReheatComp);
+    EXPECT_EQ("ZONE 1 REHEAT COIL", thisAirTerminal.ReheatName);
+    EXPECT_EQ("FAN:SYSTEMMODEL", thisAirTerminal.FanType);
+    EXPECT_EQ("ZONE 1 VS FAN", thisAirTerminal.FanName);
+    EXPECT_EQ(thisAirTerminal.ZoneMinAirFrac, 0.05);
+    EXPECT_EQ(thisAirTerminal.MaxAirVolFlowRate, 1.0);
+    // test 1: 0.05 fraction damper position
+    Real64 SysMinMassFlowRes = 1.0 * DataEnvironment::StdRhoAir * 0.05; 
+    Real64 SysMaxMassFlowRes = 1.0 * DataEnvironment::StdRhoAir;
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(1).RemainingOutputRequired = 0.0;
+    DataLoopNode::Node(InletNodeNum).MassFlowRate = SysMaxMassFlowRes;
+    DataLoopNode::Node(InletNodeNum).MassFlowRateMaxAvail = SysMaxMassFlowRes;
+    CurDeadBandOrSetback.allocate(1);
+    CurDeadBandOrSetback(1) = false;
+    DataGlobals::BeginEnvrnFlag = true;
+    FirstHVACIteration = true;
+    SingleDuct::InitSys(SysNum, FirstHVACIteration);
+    DataGlobals::BeginEnvrnFlag = false;
+    FirstHVACIteration = false;
+    SingleDuct::InitSys(SysNum, FirstHVACIteration);
+    SingleDuct::SimVAVVS(SysNum, FirstHVACIteration, ZoneNum, ZoneNodeNum);
+    // check inputs and calculated values for zone air fraction 0.05
+    EXPECT_EQ(0.05,thisAirTerminal.ZoneMinAirFrac);  // user input
+    EXPECT_EQ(0.05, thisAirTerminal.DamperPosition);
+    EXPECT_EQ(SysMinMassFlowRes, thisAirTerminal.AirMassFlowRateMax * thisAirTerminal.ZoneMinAirFrac);
+    EXPECT_EQ(SysMinMassFlowRes, thisAirTerminalOutlet.AirMassFlowRate);
+
+    // test 2: 0.10 fraction damper position
+    thisAirTerminal.ZoneMinAirFrac = 0.10; // modified user input
+    SysMinMassFlowRes = 1.0 * DataEnvironment::StdRhoAir * 0.10; 
+    DataZoneEnergyDemands::ZoneSysEnergyDemand(1).RemainingOutputRequired = 0.0;
+    DataLoopNode::Node(InletNodeNum).MassFlowRate = SysMaxMassFlowRes;
+    DataLoopNode::Node(InletNodeNum).MassFlowRateMaxAvail = SysMaxMassFlowRes;
+    DataGlobals::BeginEnvrnFlag = true;
+    FirstHVACIteration = true;
+    SingleDuct::InitSys(SysNum, FirstHVACIteration);
+    DataGlobals::BeginEnvrnFlag = false;
+    FirstHVACIteration = false;
+    SingleDuct::InitSys(SysNum, FirstHVACIteration);
+    SingleDuct::SimVAVVS(SysNum, FirstHVACIteration, ZoneNum, ZoneNodeNum);
+    // check inputs and calculated values for zone air fraction 0.10
+    EXPECT_EQ(0.10, thisAirTerminal.ZoneMinAirFrac);
+    EXPECT_EQ(0.10, thisAirTerminal.DamperPosition);
+    EXPECT_EQ(SysMinMassFlowRes, thisAirTerminal.AirMassFlowRateMax * thisAirTerminal.ZoneMinAirFrac);
+    EXPECT_EQ(SysMinMassFlowRes, thisAirTerminalOutlet.AirMassFlowRate);
+}
+
```

Comment:
The unit test checks user inputs and the damper position at the min flow rate. prior to the fix, the thisAirTerminal.DamperPosition variable was always zero.

Reply:
:+1: Great unit test.